noxfile.py

```diff
@@ -27,6 +28,8 @@
     "docs/source/conf.py",
 ]
 
+options.sessions = ["format_fix", "test", "sphinx"]
+
```

Comment:
What is this for?

Reply:
Those are the default sessions to run when doing `nox` alone. This will allow people to quickly check all pipelines before they push